src/Utilities/src/MoistThermodynamics.jl

```diff
@@ -216,32 +217,20 @@ function latent_heat_generic(T, LH_0, cp_diff)
 
 end
 
-"""
-    sat_vapor_press_liquid(T)
+abstract type Phase end
+struct Liquid <: Phase end
+struct Ice <: Phase end
 
+"""
+    `sat_vapor_pressure(T, Liquid())`
```

Comment:
were you going to rename the function?

Reply:
Yes, sorry, I misread and only extended `pres` to `pressure`. Updating now.